File: 260-single-number-iii/single-number-iii.cpp
class Solution {
public:
    vector<int> singleNumber(vector<int>& nums) {
        set<int>set;
        for(int i=0;i<nums.size();i++){
            if(set.find(nums[i])==set.end())
                set.insert(nums[i]);
            else
                set.erase(nums[i]);
        }
        vector<int>ans;
        for(auto it:set)
        {
            ans.push_back(it);
        }
        return ans;
    }
};

File: 260-single-number-iii/single-number-iii.py
class Solution:
    def singleNumber(self, nums: List[int]) -> List[int]:
        num_set = set()

        for num in nums:
            if num in num_set:
                num_set.remove(num)
            else:
                num_set.add(num)
        return list(num_set)